### PR TITLE
chore: add logos for new pools

### PR DESCRIPTION
**Pull request by dudesahn**
## 📑 Description

- add logo for new MOG-WETH & YFI-wstETH aerodrome factory vault pool, also re-use same dgnETH-ETH+ logo from old pool address for new pool

---

## ✅ Checks

-   [x] I created a new folder with the token address, **all in lowercase**
-   [x] I added the token's logo as a `32x32` PNG file, named `logo-32.png`
-   [x] I added the token's logo as a `128x128` PNG file, named `logo-128.png`
-   [x] I added the token's logo as a SVG file, named `logo.svg`
-   [x] My SVG logo is a proper SVG and not some base64 stuff
-   [x] My documentation/website clearly display the token address somewhere

### CODE REVIEW

**Comment by dudesahn** [comment]
oof, seems like this image is base64...reached out to mog team to try and get a proper SVG version

File: tokens/8453/0x4a311ac4563abc30e71d0631c88a6232c1309ac5/logo.svg [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
...GPNmv4Ly+fx127W9gdreHa0hm88dWP4FvO/fAhuDd6zKY4TGRxsFXsZMZDsRL4Gq+Tq/UaF4Jbvki0tRuxZUDsLtHwg81vevZTwQg328+bGzz+5GN/6VcjYf40utSuzj6Ef5dp8fBXcoCsPMMAGYLF+l+RrQf6i/mfiocoEWv+fCj/yfRuPHH2ZbzzrR/GQ/OncGHg5HBZCUV7rHU7WGt6nGgYYb2PSd/jAK3AI5OPnOzIQHZY7ue4Mmpwy2JPqIXS80Hb4sRiT+UImRCc4JElCafdGLPRQsdbme2gaRnFvUA7n2GZPsfxLpo5K8tcwnJ/EeP+kkAA2U+lkmyHiE8CSoI+AkrK1gl6iXRLcBDAGXaD98bAm4AqMRXWS8IyCrwZZAQMJUCGPyPBhimTr2beYNT2WkSLBJ1Bn8jNWj42CA0wi8dOoDWTb+RzA7iSfsefCQs5bJdt/DbQ04LsoJyA4ZwrIJN/0y8aj2Vk9NpHmXRBJfdjWLmY3DOz86erumjer3yRafvad6hddiRJXdBh0IJS8VhmjvQdz6k+6udFoKsqJbFieEOhdEwx1sfgX3mk0lbKcWcgqHKUBqaR2Mgs6trtz4ysxr/3pgM4OP6rueH+7M83+KvPn8E9a11h0V7rJBxAyHvdn88xOzhAN5+inx8MAWlCjlUVI3psR8toxstol1awPJlgMhkNQDJg8zVeFNtiE2Ns7i7wA295Hr/rLcC1BlidVNHT9t4lvQjbmBHMkoLr4CYDmPhKxegVmt6scPyElV+ZbJ48wukHYT6rzxYp0H07jFoYNjWD+4eeb+VvzGYyJRMDfMtmh90hzJgnAHk4DYDC2snH7fsugTO+xuRm1D3fkOdRtoxYAdxmYTdz/XxfQV0GnAGPST8U36SuM4A6z776zrAjG8Z/iRKPEmBAe53vsF47Al6dJUIspqO1k5qnADQPnGwCBPAiyXqsauiaNdQYD3h0u2bM8+XrAKbRVPyrmhMjj1vx0GbbbOXwXW7IhhPYdaMUWkwCrtcMHNXH+AwcpEV6jswjrSoHk+H3XdyOWftWzPtbsNeu4lp7Cjs9czFP9Pdnmw387osvqbZ1s7SEz4zegIc3fxn/0Ys/OiQkr4BjWMey7LuN0o6al7QxvgVdf8VkR4JmKE9Tu7t4XdCMUviUajM8IKOyiRvYyVaw33z0/tX34dnXODV81X
```
Comment (abmisx0) [inline tokens/8453/0x4a311ac4563abc30e71d0631c88a6232c1309ac5/logo.svg:31]: ser it's base64 encoded, I can't merge it as is

**Comment by abmisx0** [comment]
> oof, seems like this image is base64...reached out to mog team to try and get a proper SVG version

if you get a proper one, can you submit it for their token as well in this PR

**Comment by abmisx0** [comment]
our designer came through and made an SVG, see #443

**Comment by dudesahn** [comment]
amazing, ty good ser @abmisx0

**Review by abmisx0 — APPROVED** [review]

**Pull request merged**